invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -114,17 +125,37 @@ <h1>{{ metadata.title }}</h1>
         {%- block record_files -%}
         {%- if files is not none -%}
           {%- if files.enabled and record|can_list_files %}
-            {%- set default_preview = files.default_preview %}
```

Comment:
Was this meant to be deleted? Line 130 seems to use it.

Reply:
My bad, this shouldn't be removed, thanks!